mobile/mobile/Base.lproj/LaunchScreen.storyboard

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Were you able to use storyboard with SwfitUI? For some reason I thought using SwfitUI meant you couldn't use storyboards.

Reply:
Yeah, I did not use storyboards at all. That is kind of my understanding too, that SwiftUI replaces storyboards. This was just an auto-generated file that was in the repo and has not been editing since I started the project.